services/tests/factories.py

```diff
@@ -19,3 +21,11 @@ class ServiceConnectionFactory(factory.django.DjangoModelFactory):
 
     class Meta:
         model = ServiceConnection
+
+
+class AllowedDataFieldFactory(factory.django.DjangoModelFactory):
+    field_name = "name"
```

Comment:
so, `factory.Faker(...)` did not work here?

Reply:
It did, but at least for the current test cases it doesn't need to be anything random and actually semantically `field_name` and `label` should stand for the same "thing".

Reply:
yes, ok, let's go with this code 👍